typegate/tests/runtimes/deno/deno_test.ts

```diff
@@ -108,54 +107,28 @@ test("Deno runtime: permissions", async (t) => {
   });
 });
 
-test("Deno runtime: reloading", async (t) => {
-  const tmpDir = join(testDir, "tmp");
```

Comment:
shouldn't we keep this test?

Reply:
The dynamic reload test creates a tmp file and its content is rewritten at runtime: this is impossible to 100% replicate in the current implementation as `scripts/deno/*` is sent to typegate then DenoRuntime imports from that.
Alternartively, we could rewrite any script at `./tmp/{hash}/*` and reproduce the reload (=script overwritten at runtime as of my understanding) but I am not quite sure what behavior it tests exactly.

Reply:
We shall manage to reproduce the same logic, by re-uploading a typegraph with a slight different function logic. That will allow us to test well the clean-up and re-setting process of the files.